app/views/smart_proxies/plugins/_dns.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="row">
+  <div class="col-md-12">
```

Comment:
This should be the default on bootstrap but I can see why you're doing it, otherwise the content is misaligned. I think this html needs to be slightly modified to look like the Overview page. Here's what I did:

```html
<div class="row">
  <h3><%= _('DNS') %></h3>
</div>
<%= render :partial => 'smart_proxies/plugins/plugin_version', :locals => {:feature => feature, :proxy => @smart_proxy } %>
<div class="row">
  <div class="col-md-4">
    <%= _('Domains') %>
  </div>
  <div class="col-md-8">
    <%= generate_links_for(@smart_proxy.domains).html_safe %>
  </div>
</div>
```
That's the same layout as in the Overview page, now it looks a little different.
![screenshot from 2015-12-28 16-29-29](https://cloud.githubusercontent.com/assets/598891/12020814/816e2a62-ad80-11e5-990a-0fc138003156.png)